samples/templates/default-azuredeploy-sql.json

```diff
@@ -143,7 +151,8 @@
             "FhirServer:Security:Authentication:Audience": "[parameters('securityAuthenticationAudience')]",
             "CosmosDb:ContinuationTokenSizeLimitInKb": "1",
             "SqlServer:Initialize": "true",
-            "DataStore": "SqlServer"
+            "DataStore": "SqlServer",
+            "PROJECT": "[concat('src/Microsoft.Health.Fhir.', parameters('fhirVersion'),'.Web/Microsoft.Health.Fhir.', parameters('fhirVersion'), '.Web.csproj')]"
```

Comment:
why is PROJECT uppercase?

Reply:
Setting is here: https://github.com/projectkudu/kudu/wiki/Customizing-deployments#using-app-settings-instead-of-a-deployment-file

Reply:
Now merged if needed in `combinedFhirServerConfigProperties`